galaxy_ng/app/access_control/access_policy.py

```diff
@@ -239,11 +241,17 @@ class UserAccessPolicy(AccessPolicyBase):
     NAME = 'UserViewSet'
 
     def user_is_superuser(self, request, view, action):
-        user = view.get_object()
+        if view.kwargs.get("pk"):
+            user = view.get_object()
+        else:
+            user = request.user
         return user.is_superuser
 
     def is_current_user(self, request, view, action):
-        return request.user == view.get_object()
+        if view.kwargs.get("pk"):
+            return request.user == view.get_object()
+
+        return True
```

Comment:
Should this default to True? Feels like it should default False until proven that `request.user == view.get_object()` ?

But I'm not sure how this get's called with no pk, since it is only used from UserViewSet delete and I'm making the assumption that DELETE is only valid method in /_ui/users/<pk>/ .